flowmachine/tests/test_subscriber_distance_counterparts.py

```diff
@@ -0,0 +1,118 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+Tests for the DistanceCounterparts().
+"""
+
+
+from flowmachine.features.subscriber import DistanceCounterparts
+from flowmachine.features.utilities import EventsTablesUnion
+from flowmachine.features.spatial.distance_matrix import DistanceMatrix
+
+import pytest
+
+
+@pytest.fixture()
+def distance_matrix(get_dataframe):
+    """ Calculates the distance matrix between cells. """
+    return get_dataframe(DistanceMatrix())
+
+
+@pytest.fixture()
+def distance_counterparts(get_dataframe, distance_matrix):
```

Comment:
This is a reference implementation using pandas, right? Can we make this clear in the fixture name, and also explain it in the docstring? E.g. name the fixture `distance_counterparts_expected` or `distance_counterparts_reference` or similar?